draco/asp/define.lp

```diff
@@ -47,3 +47,9 @@ domain((encoding,stack),(zero;center;normalize)).
 
 % @definition(scale_zero) Scale start with 0.
 domain((scale,zero),true).
+
+% @definition(facets) Definition for facets
+domain((facet,field),F) :- entity(field,_,F).
+domain((facet,channel),(row;col)).
+domain((facet,binning),(10;25;200)).
```

Comment:
Extract the binning options into something we can reuse for the encoding binning.

Reply:
Do you mean to add a helper to know if a field is binned or not?

Reply:
No, add a fact with `binCounts()` or something so we can reuse the possible bin counts.